www/.gitignore

```diff
@@ -34,3 +34,14 @@ src/cache
 
 # Env variables
 .env*
+
+# Dependency directories
+node_modules/
```

Comment:
this should be handled by top level .gitignore: https://github.com/gatsbyjs/gatsby/blob/f671f8632c8253eca4208b9aafd13bf89799e990/.gitignore#L32 so we should be able to remove that

Reply:
how do i edit my pull request ?